UPGRADE-2.0.md

```diff
@@ -1,6 +1,36 @@
 UPGRADE FROM 1.x to 2.0
 =======================
 
+### SonataAdmin Support
+
+ * The Admin extension to edit seo metadata was moved into `symfony-cmf/sonata-admin-integration`.
+   The change of the namespace is handled in the service definition. So you have to change the usage of the
+   service.
+   
+   Before:
+   
+   ```yaml
+        # app/config/config.yml
+        
+        sonata_admin:
```

Comment:
rollback. core-bundle and menu-bundle simply add their own information to the extension. The example i gave is the enalbling of the extension, but it does not look good at all.